src/server/app.ts

```diff
@@ -111,6 +111,30 @@ app.post("/api/score", async (req, res) => {
       surrender,
     });
 
+    app.post("/api/quit", async (req, res) => {
```

Comment:
It looks like this `app.post` is inside another `app.post`?
This can't be right :)